test/ReportPortal.Shared.Tests/Reporter/ReporterTest.cs

```diff
@@ -54,6 +54,28 @@ public void ShouldFulfillLaunchInfo()
             Assert.NotNull(launchReporter.Info.Url);
         }
 
+        [Fact]
+        public void ShouldFulfillTestInfo()
+        {
+            var now = DateTime.UtcNow;
+
+            var service = new MockServiceBuilder().Build();
+
+            var launchScheduler = new LaunchReporterBuilder(service.Object);
+            var launchReporter = launchScheduler.Build(1, 0, 0);
+
+            launchReporter.Sync();
+
+            var testReporter = launchReporter.ChildTestReporters[0];
+
+            Assert.NotNull(testReporter);
```

Comment:
I suggest to use only FluentAssestion assertion method for new tests to don't mix assertion strategies? What do you think?

Reply:
Fixed.